docs/transports.md

```diff
@@ -223,6 +223,22 @@ logger.info('hello world')
 __NOTE: there is no "default" destination for a pipeline but
 a terminating target, i.e. a `Writable` stream.__
 
+### TypeScript compatibility
+
+Pino provides basic support for transports written in TypeScript.
+
+Ideally, they should be transpiled to ensure maximum compatibility, but some
+times you might want to use tools such as TS-Node or TSM, to execute your
```

Comment:
```suggestion
times you might want to use tools such as TS-Node or TSM, to execute your
```

Why mention "or TSM" if it isn't actually supported?

Reply:
dropped the reference.